core/src/net/buffers/mod.rs

```diff
@@ -91,6 +92,14 @@ impl BufferConfig {
         buffer_config
     }
 
+    pub fn from_config_file<P>(path: P) -> BufferConfig
```

Comment:
This is missing docs. 

How did the compiler even let this pass? I thought I set it to reject public functions without documentation?

Reply:
Hmm the compiler didn't tell my anything.